### PR TITLE
 BrenKt-SK1128-TRANS-WEB-Email-locked_for_spam 

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

File: pages/web/emails/locked_for_spam/guide.es-es.md [hunk 1]
```diff
@@ -1,52 +1,88 @@
...
-
-Inicie sesión en el [área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es) y acceda a la sección `Webcloud`{.action}. Haga clic en `Correos electrónicos`{.action} y seleccione el nombre de dominio correspondiente.
-
-Acceda a la pestaña `Cuentas de correo electrónico`{.action} de su plataforma. Si la columna «Estado» —a la derecha de la dirección de correo electrónico correspondiente— indica «Correo no deseado», haga clic en esa opción y, seguidamente, en `Responder al tique`{.action}. Seguidamente, continúe con el [paso 2](./#paso-2-acceder-al-tique-de-asistencia_1){.external} de la guía.
-
-![spam](images/blocked-for-SPAM-01-03.png){.thumbnail}
-
-
-### Paso 2: acceder al tique de asistencia
+- Haber iniciado sesión en el [área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es) y accedido a la sección `Webcloud`{.action}.
```
Comment (OVH-Team-Guides) [inline pages/web/emails/locked_for_spam/guide.es-es.md]:
```suggestion
- Haber iniciado sesión en el [área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es) y accedido a la sección `Web Cloud`{.action}.
```

File: pages/web/emails/locked_for_spam/guide.es-es.md [hunk 2]
```diff
@@ -1,52 +1,88 @@
...
+
+## Procedimiento <a name="instructions"></a>
+
+Antes de continuar, y si el bloqueo afecta a una dirección de correo electrónico de tipo MXplan, identifique la versión que tiene para consultar el progreso del proceso de desbloqueo. Compruebe cómo distinguir las dos versiones utilizando la siguiente tabla.
+
+|Versión histórica de la solución MX Plan|Nueva versión de la solución MX Plan|
+|---|---|
+|![email](images/mxplan-starter-legacy-step1.png){.thumbnail}< br> Consulte la oferta en el cuadro "Suscripción"|[email](images/mxplan-starter-new-step1.png){.thumbnail}< br>Consulte la referencia del servidor en el recuadro "Resumen"|
```
Comment (OVH-Team-Guides) [inline pages/web/emails/locked_for_spam/guide.es-es.md]:
```suggestion
|![email](images/mxplan-starter-legacy-step1.png){.thumbnail}< br> Consulte la oferta en el cuadro "Suscripción"|![email](images/mxplan-starter-new-step1.png){.thumbnail}< br>Consulte la referencia del servidor en el recuadro "Resumen"|
```

File: pages/web/emails/locked_for_spam/guide.es-es.md [hunk 3]
```diff
@@ -1,52 +1,88 @@
...
+
+En primer lugar, asegúrese de que los usuarios de la dirección de correo electrónico bloqueada no son los causantes directamente del bloqueo, debido a un uso inusual de la dirección de correo electrónico (por ejemplo, tras el envío masivo de mensajes de correo). En ese caso, debe corregir la situación antes de desbloquear la dirección.
+
+Si la actividad sospechosa detectada por el antispam no ha sido iniciada por el usuario legítimo o los usuarios de la dirección de correo electrónico, tome las siguientes medidas necesarias y detalladas:
+
+- Realice un análisis antivirus de cada uno de los equipos que utilicen la dirección de correo electrónico bloqueada para spam y aplique un parche si estos están infectados.
+
+- Compruebe todos los programas que utilicen las claves de la dirección de correo electrónico bloqueada para spam (p. ej.: copiadora, software de gestión empresarial y de mensajería).
+
+### Paso 2: comprobar el estado de la dirección de correo electrónico y acceder al tique de asistencia asociado <a name="step2"></a>
```
Comment (OVH-Team-Guides) [inline pages/web/emails/locked_for_spam/guide.es-es.md]:
```suggestion
### Etapa 2: comprobar el estado de la dirección de correo electrónico y acceder al tique de asistencia asociado <a name="step2"></a>
```

File: pages/web/emails/locked_for_spam/guide.es-es.md [hunk 4]
```diff
@@ -1,52 +1,88 @@
...
+
+Seleccione el servicio de correo en las siguientes pestañas:
+
+> [!tabs]
+> **Exchange**
+>>
+>> Inicie sesión en el [área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es) y acceda a la sección `Webcloud`{.action}. Haga clic en `Microsoft`{.action}, luego en `Exchange`{.action} y, por último, seleccione la plataforma Exchange correspondiente.
```
Comment (OVH-Team-Guides) [inline pages/web/emails/locked_for_spam/guide.es-es.md]:
```suggestion
>> Inicie sesión en el [área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es) y acceda a la sección `Web Cloud`{.action}. Haga clic en `Microsoft`{.action}, luego en `Exchange`{.action} y, por último, seleccione la plataforma Exchange correspondiente.
```

File: pages/web/emails/locked_for_spam/guide.es-es.md [hunk 5]
```diff
@@ -1,52 +1,88 @@
...
+>>
+>> ceda a la pestaña `Cuentas de correo electrónico`{.action} de su plataforma. Si la columna «Estado» de la dirección de correo electrónico correspondiente indica «bloqueado», haga clic en `...`{.action} a la derecha de la cuenta y, seguidamente, en `Desbloquear`{.action}.El desbloqueo de la dirección de correo electrónico no se realiza automáticamente. Es necesario contactar con el soporte en el tíquet de asistencia respondiendo a las 3 preguntas formuladas.
```
Comment (OVH-Team-Guides) [inline pages/web/emails/locked_for_spam/guide.es-es.md]:
```suggestion
>> Acceda a la pestaña `Cuentas de correo electrónico`{.action} de su plataforma. Si la columna «Estado» de la dirección de correo electrónico correspondiente indica «bloqueado», haga clic en `...`{.action} a la derecha de la cuenta y, seguidamente, en `Desbloquear`{.action}. El desbloqueo de la dirección de correo electrónico no se realiza automáticamente. Es necesario contactar con el soporte en el tíquet de asistencia respondiendo a las 3 preguntas formuladas.
```

File: pages/web/emails/locked_for_spam/guide.es-es.md [hunk 6]
```diff
@@ -1,52 +1,88 @@
...
+>> Pasar al [paso 3](#step3) de la guía.
+>>
+>> ![spam](images/blocked-for-SPAM-01-01.png){.thumbnail}
+>> 
+> **Email Pro**
+>>
+>> Inicie sesión en el [área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es) y acceda a la sección `Webcloud`{.action}. Haga clic en `Email Pro`{.action} y, seguidamente, seleccione la plataforma Email Pro correspondiente.
```
Comment (OVH-Team-Guides) [inline pages/web/emails/locked_for_spam/guide.es-es.md:54]:
```suggestion
>> Inicie sesión en el [área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es) y acceda a la sección `Web Cloud`{.action}. Haga clic en `Email Pro`{.action} y, seguidamente, seleccione la plataforma Email Pro correspondiente.
```

File: pages/web/emails/locked_for_spam/guide.es-es.md [hunk 7]
```diff
@@ -1,52 +1,88 @@
...
+>>
+>> Acceda a la pestaña `Cuentas de correo electrónico`{.action} de su plataforma. Si la columna «Estado» a la derecha de la dirección de correo electrónico correspondiente indica «Correo no deseado», haga clic en esa opción y, seguidamente, en `Responder al tique`{.action}. El desbloqueo de la dirección de correo electrónico no se realiza automáticamente. Es necesario contactar con el soporte en el tíquet de asistencia respondiendo a las 3 preguntas formuladas.
```
Comment (OVH-Team-Guides) [inline pages/web/emails/locked_for_spam/guide.es-es.md:56]:
```suggestion
>> Acceda a la pestaña `Cuentas de correo electrónico`{.action} de su plataforma. Si la columna «Estado» a la derecha de la dirección de correo electrónico correspondiente indica «Correo no deseado», haga clic en esa opción y, seguidamente, en `Responder al tiquet`{.action}. El desbloqueo de la dirección de correo electrónico no se realiza automáticamente. Es necesario contactar con el soporte en el tíquet de asistencia respondiendo a las 3 preguntas formuladas.
```

File: pages/web/emails/locked_for_spam/guide.es-es.md [hunk 8]
```diff
@@ -1,52 +1,88 @@
...
+>> Pasar al [paso 3](#step3) de la guía.
```
Comment (OVH-Team-Guides) [inline pages/web/emails/locked_for_spam/guide.es-es.md:57]:
```suggestion
>> Continúe con el [paso 3](#step3) de la guía.
```

File: pages/web/emails/locked_for_spam/guide.es-es.md [hunk 9]
```diff
@@ -1,52 +1,88 @@
...
+
+Seleccione el servicio de correo en las siguientes pestañas:
+
+> [!tabs]
+> **Exchange**
+>>
+>> Inicie sesión en el [área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es) y acceda a la sección `Webcloud`{.action}. Haga clic en `Microsoft`{.action}, luego en `Exchange`{.action} y, por último, seleccione la plataforma Exchange correspondiente.
+>>
+>> ceda a la pestaña `Cuentas de correo electrónico`{.action} de su plataforma. Si la columna «Estado» de la dirección de correo electrónico correspondiente indica «bloqueado», haga clic en `...`{.action} a la derecha de la cuenta y, seguidamente, en `Desbloquear`{.action}.El desbloqueo de la dirección de correo electrónico no se realiza automáticamente. Es necesario contactar con el soporte en el tíquet de asistencia respondiendo a las 3 preguntas formuladas.
+>> Pasar al [paso 3](#step3) de la guía.
```
Comment (OVH-Team-Guides) [inline pages/web/emails/locked_for_spam/guide.es-es.md]:
```suggestion
>> Continúe con el [paso 3](#step3) de la guía.
```

File: pages/web/emails/locked_for_spam/guide.es-es.md [hunk 10]
```diff
@@ -1,52 +1,88 @@
...
+>>
+>> ![spam](images/blocked-for-SPAM-01-02.png){.thumbnail}
+>>
+>  **MX plan - nueva versión**
+>>
+>> Inicie sesión en el [área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es) y acceda a la sección `Webcloud`{.action}. Haga clic en `Correos electrónicos`{.action} y seleccione el nombre de dominio correspondiente.
+>>
+>> Acceda a la pestaña `Cuentas de correo electrónico`{.action} de su plataforma. Si la columna «Estado» —a la derecha de la dirección de correo electrónico correspondiente— indica «Correo no deseado», haga clic en esa opción y, seguidamente, en `Responder al tique`{.action}. El desbloqueo de la dirección de correo electrónico no se realiza automáticamente. Es necesario contactar con el soporte en el tíquet de asistencia respondiendo a las 3 preguntas formuladas.
+>> Pasar al [paso 3](#step3) de la guía.
```
Comment (OVH-Team-Guides) [inline pages/web/emails/locked_for_spam/guide.es-es.md]:
```suggestion
>> Continúe con el [paso 3](#step3) de la guía.
```

File: pages/web/emails/locked_for_spam/guide.es-es.md [hunk 11]
```diff
@@ -1,52 +1,88 @@
...
+>> Inicie sesión en el [área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es) y acceda a la sección `Webcloud`{.action}. Haga clic en `Email Pro`{.action} y, seguidamente, seleccione la plataforma Email Pro correspondiente.
+>>
+>> Acceda a la pestaña `Cuentas de correo electrónico`{.action} de su plataforma. Si la columna «Estado» a la derecha de la dirección de correo electrónico correspondiente indica «Correo no deseado», haga clic en esa opción y, seguidamente, en `Responder al tique`{.action}. El desbloqueo de la dirección de correo electrónico no se realiza automáticamente. Es necesario contactar con el soporte en el tíquet de asistencia respondiendo a las 3 preguntas formuladas.
+>> Pasar al [paso 3](#step3) de la guía.
+>>
+>> ![spam](images/blocked-for-SPAM-01-02.png){.thumbnail}
+>>
+>  **MX plan - nueva versión**
+>>
+>> Inicie sesión en el [área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es) y acceda a la sección `Webcloud`{.action}. Haga clic en `Correos electrónicos`{.action} y seleccione el nombre de dominio correspondiente.
```
Comment (OVH-Team-Guides) [inline pages/web/emails/locked_for_spam/guide.es-es.md]:
```suggestion
>> Inicie sesión en el [área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es) y acceda a la sección `Web Cloud`{.action}. Haga clic en `Correos electrónicos`{.action} y seleccione el nombre de dominio correspondiente.
```

File: pages/web/emails/locked_for_spam/guide.es-es.md [hunk 12]
```diff
@@ -1,52 +1,88 @@
...
+>>
+>> Acceda a la pestaña `Cuentas de correo electrónico`{.action} de su plataforma. Si la columna «Estado» —a la derecha de la dirección de correo electrónico correspondiente— indica «Correo no deseado», haga clic en esa opción y, seguidamente, en `Responder al tique`{.action}. El desbloqueo de la dirección de correo electrónico no se realiza automáticamente. Es necesario contactar con el soporte en el tíquet de asistencia respondiendo a las 3 preguntas formuladas.
```
Comment (OVH-Team-Guides) [inline pages/web/emails/locked_for_spam/guide.es-es.md]:
```suggestion
>> Acceda a la pestaña `Cuentas de correo electrónico`{.action} de su plataforma. Si la columna «Estado» —a la derecha de la dirección de correo electrónico correspondiente— indica «Correo no deseado», haga clic en esa opción y, seguidamente, en `Responder al tiquet`{.action}. El desbloqueo de la dirección de correo electrónico no se realiza automáticamente. Es necesario contactar con el soporte en el tíquet de asistencia respondiendo a las 3 preguntas formuladas.
```

File: pages/web/emails/locked_for_spam/guide.es-es.md [hunk 13]
```diff
@@ -1,52 +1,88 @@
...
+>> Pasar al [paso 3](#step3) de la guía.
+>>
+>> ![spam](images/blocked-for-SPAM-01-03.png){.thumbnail}
+>>
+>  **MX plan - histórico**
+>>
+>> Si el bloqueo afecta a una dirección de correo electrónico [MXplan versión histórica](#instructions), no hay tíquet de soporte. Por favor, consulte el [paso 1](#step1) de esta guía antes de seguir las instrucciones siguientes.
```
Comment (OVH-Team-Guides) [inline pages/web/emails/locked_for_spam/guide.es-es.md]:
```suggestion
>> Si el bloqueo afecta a una dirección de correo electrónico [MXplan versión histórica](#instructions), no hay tíquet de soporte. Por favor, consulte la [etapa 1](#step1) de esta guía antes de seguir las instrucciones siguientes.
```

**Review by OVH-Team-Guides — COMMENTED** [review]
ES review only so far
Please add our usual Systran translation warnings

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**